assets/Standard.frag

```diff
@@ -0,0 +1,9 @@
+#version 450 core
+
+layout(location = 0) in vec3 fPos;
+
+layout(location = 0) out vec4 Color;
```

Comment:
Keep naming consistent here. Lines 3 and 5 have different casing styles. IMO, shader variables should have their use in front of them. I always use `a_` for attributes, `u_` for uniforms and `o_` for output. Just a suggestion.